.github/workflows/playwright.yml

```diff
@@ -1,29 +1,102 @@
 name: Playwright Tests
 on:
   push:
-    branches: [ main, master ]
+    branches: [ main ]
   pull_request:
-    branches: [ main, master ]
+    branches: [ main ]
 jobs:
   test:
-    timeout-minutes: 60
+    timeout-minutes: 30
     runs-on: ubuntu-latest
     # fow now, skip it
-    if: false
+    # if: false
     steps:
     - uses: actions/checkout@v3
+
+    # install node stuff
     - uses: actions/setup-node@v3
       with:
-        node-version: 18
-    - name: Install dependencies
-      run: npm ci
+        node-version: 20
+        cache: 'npm'
+    - name: Install NPM dependencies
+      run: |
+        make e2e-install
+
+    # install python stuff
+    - name: Setup python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.8'
+        cache: 'pip'
+    - name: Install Python dependencies
+      run: |
+        make dev-install-dependencies-pack
+        make dev-install-dependencies-pack-sudo
+        make dev-install-dependencies-for-development
+
+    # build splunk application
+    - name: Build Splunk Application
+      run: |
+        scripts/pack.sh \
+          --version "9.9.9" \
+          --input TA_dataset \
+          --output output \
+          --release release
+
+    - name: Fix Splunk Application
+      run: |
+        make dev-fix-package
+
+    # run Splunk
+    - name: Run Splunk
+      run: |
+        make docker-splunk-run-ci
+
+    # we are fine with installing only chromium
     - name: Install Playwright Browsers
-      run: npx playwright install --with-deps
+      run: npx playwright install chromium --with-deps
```

Comment:
I wonder if we need this, or if it should be part of `package.json`. Feels strange to run `npm install` and then install some more stuff.

Reply:
This was generated by the playwright - https://playwright.dev/docs/ci-intro - therefore I have kept it.

Reply:
Right, they have it in the docs. I don't see we would be doing this in the dataset-webapp repo, which made me think it's not needed. Anyway, feel free to keep it.